src/java/org/apache/cassandra/concurrent/SEPExecutor.java

```diff
@@ -35,16 +35,13 @@
 
     public final int maxWorkers;
     public final String name;
-    private final int maxTasksQueued;
     private final SEPMetrics metrics;
 
     // stores both a set of work permits and task permits:
     //  bottom 32 bits are number of queued tasks, in the range [0..maxTasksQueued]   (initially 0)
     //  top 32 bits are number of work permits available in the range [0..maxWorkers]   (initially maxWorkers)
     private final AtomicLong permits = new AtomicLong();
 
-    // producers wait on this when there is no room on the queue
-    private final WaitQueue hasRoom = new WaitQueue();
```

Comment:
Nit: This leaves a unused import of `WaitQueue`

Reply:
I will fix it on commit.